reference/speeduino.ini

```diff
@@ -275,8 +275,8 @@ page = 1
 
       reqFuel       = scalar, U08,      24,        "ms",        0.1,       0.0,   0.0,    25.5,      1
       divider       = scalar, U08,      25,        "",          1.0,       0.0
-      alternate     = bits,   U08,      26, [0:0], "Simultaneous", "Alternating"
-      multiplyMAP1   = bits,   U08,      26, [1:1], "No",       "Yes"
+      injTiming     = bits,   U08,      26, [0:0], "Simultaneous", "Alternating"
```

Comment:
Unfortunately `alternate` is a reserved variable name used by TunerStudio. Changing this breaks the `req_fuel` calculator and so will need to be left the same.

Reply:
Fixed